### DISCUSSION TITLE
Some question about scaling_factor

### DISCUSSION THREAD

**Discussion by fighting-xq** [NVlabs/sionna#619] [Q&A] [answered]
I have some questions about the scaling_factor in [diff_rt](https://github.com/NVlabs/diff-rt-calibration). Why you want to use scaling_factor and why you get scaling_factor like this
```py
scaling_factor.assign(delta*scaling_factor + (1-delta)*mse_power_scaling_factor(h_rt, h_meas))
```

I know this  is some question about [diff_rt](https://github.com/NVlabs/diff-rt-calibration) but I didn't find any place to draw a question.

**Comment by jhoydis** [comment, marked as answer]
Hi @qiangdadi,

Did you have a look at the corresponding [paper](https://arxiv.org/abs/2311.18558)? It is explained there.
The problem is that our dataset has a common but unknown scaling factor.
We compute this factor iteratively via a moving average. This is the line you have pasted.
